src/styles/partners.scss

```diff
@@ -1,24 +1,39 @@
+@import "_vars.scss";
+
 .partners-page {
-  .hero {
-    .hero-body {
+  .jf-partner-card {
+    height: 100%;
+    .has-background-white {
+      border: 1px solid $justfix-black;
+    }
+    .has-background-black {
       width: 100%;
-      max-width: 720px;
+      .eyebrow {
+        min-height: $spacing-11;
+      }
+    }
+    a {
+      overflow-wrap: anywhere;
     }
-  }
-  .logos {
-    padding-bottom: 3rem;
-    .logo {
-      padding: 2rem;
-      .image {
-        width: 150px;
-        height: 150px;
-        transition: transform 100ms ease-in-out;
-        transform-origin: center;
 
-        &:hover {
-          opacity: 0.6;
-          transform: scale(1.1);
-        }
+    &.jf-accordion-item {
```

Comment:
These seem to flash with a slightly different color for a split second when opening/closing. I noticed something similar on the navbar, and I think it was a bulma default transition - not sure if this is the same thing.